assets/base.css

```diff
@@ -3418,4 +3418,30 @@ details-disclosure > details {
     --easter-egg: none;
     --sparkle: url('./sparkle.gif');
   }
+
+  .animate--hover-raise .card-wrapper .card--card,
+  .animate--hover-raise .card-wrapper .card--standard .card__inner,
+  .animate--hover-raise .button:not(.button--tertiary),
+  .animate--hover-raise .shopify-challenge__button,
+  .animate--hover-raise .customer button,
+  .animate--hover-raise .shopify-payment-button__button,
+  .animate--hover-raise .deferred-media__poster-button {
+    transition: transform var(--duration-long) cubic-bezier(0,0,.35,1);
```

Comment:
I wonder if for this one we could pick the `--duration-default` instead of the long one. That could make this option snappier!

Reply:
I adjusted this in https://github.com/Shopify/dawn/pull/2575/commits/d3718a033d1091751519564f342afabe69fec0dc. It felt too quick for larger elements like cards, but it felt okay for buttons. So I've only applied the `--duration-default` to buttons for now. I'm not sure how I feel about it yet — let me know what you think.